src/components/SubCategoryDetails/SubCategoryDetails.scss

```diff
@@ -0,0 +1,45 @@
+@import '~style/variables';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding-top: 32px;
+  width: 100%;
+  margin: 0 0 16px;
+  background: $color-white;
+  padding-left: 16px;
+  overflow: hidden;
+
+  @include for-tablet-landscape-up {
+    padding-left: 32px;
+  }
+}
+
+.heading {
+  display: flex;
+  align-items: center;
+  width: 100%;
```

Comment:
Is there a reason for `width: 100%`? Anything with a display of `block` or `flex` will be a 'block-level' element which will take all available width.

Unless there's a browser quirk I'm missing.